process-engine/process_engine_meta
# 📦 Implement new styleguide and build system

## Changes

1. Remove gulp and switch to a purely `tsc` based build system.
2. Replace tslint with eslint.
3. Apply the new styleguide defined in the eslint ruleset.
    - You can find the ruleset [here](https://github.com/essential-projects/eslint-config/blob/develop/index.js)
4. Update template files.

## Issues

PR: #196

## How to test the changes

Run the following commands:
- `npm install`
- `npm run build`
- `npm run lint`
- `npm run test`

All changes made affect code-styling only. So all should work as before.
